spyderlib/qt/__init__.py

```diff
@@ -57,6 +54,17 @@
             API_NAME += (" (API v%d)" % sip.getapi('QString'))
         except AttributeError:
             pass
+        from PyQt4 import uic  # analysis:ignore
```

Comment:
Ok, no problem. I just wanted to know why you decided to add it given that we don't use QtDesigner in Spyder.

By the way, we're planning to move to use [qtpy](https://github.com/spyder-ide/qtpy) (one of the Spyder org projects) as our Qt shim.

I think that would help your projects too, to avoid copy/pasting the Spyder shim around, and to have everything consolidated in one place.